### PR TITLE
fix(build): fix slf4j import/exclusion, and remove duplicate trove4j dep

### LINKED ISSUE #26: slf4j binding warning
As this project is a library the [logging dependencies](https://github.com/conveyal/gtfs-lib/blob/master/pom.xml#L128-L132) should be set to 'test' or 'runtime' scope to avoid binding the wrong slf4j binding and this warning: http://www.slf4j.org/codes.html#multiple_bindings

As a workaround one can use:
```xml
        <dependency>
            <groupId>com.conveyal</groupId>
            <artifactId>gtfs-lib</artifactId>
            <version>1.2.0</version>

            <exclusions>
                <exclusion> 
                    <groupId>org.slf4j</groupId>
                    <artifactId>slf4j-simple</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```

### PR DESCRIPTION
**Pull request by landonreed**
fixes #26

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,6 +86,18 @@
     </repository>
   </repositories>
   <dependencies>
+    <dependency>
+      <groupId>com.conveyal</groupId>
+      <artifactId>gtfs-lib</artifactId>
```
Comment (mattwigway) [inline pom.xml]: Why are we adding gtfs-lib as a dependency of itself?

**Comment by landonreed** [comment]
How's this @mattwigway?

**Comment by landonreed** [comment]
How's this @mattwigway?

**Comment by mattwigway** [comment]
This looks good to me. @karussell this look good to you?

**Comment by karussell** [comment]
Yes, thanks! This should work.

**Pull request merged**